product_docs/docs/pwr/1/index.mdx

```diff
@@ -6,15 +6,9 @@ navigation:
   - using
 ---
 
-Postgres Workload Report (PWR) is a Python-based tool used for building PostgreSQL workload reports in HTML, Markdown, DOCX, and PDF mode, mimicking the reports provided by the Automatic Workload Repository (AWR) reporting tool from Oracle.
+Postgres Workload Report (PWR) is a Python-based tool used for building PostgreSQL workload reports in HTML, Markdown, DOCX, and PDF mode. These reports mimic the reports provided by the Automatic Workload Repository (AWR) reporting tool from Oracle.
 
 Using a Postgres connection, you can execute Postgres Workload Report from any machine with access to the server for which you want a report. Postgres Workload Report uses Lasso for collecting data, so Lasso is a prerequisite. The `edb-pwr` package has an explicit dependency on `edb-lasso` being installed.
 
-On the Postgres server, you should have `edb_wait_states` version 1.2 or later installed and loaded. The extension also must be created,
-preferably on the main database, so the `edb_wait_states` functions are available for Postgres Workload Report to collect saved snapshot data.
-
-For more information about PWR, see: 
-
-  - [Installing](installing)
-  - [Configuring](configuring)
-  - [Using](using)
+On the Postgres server, you must have `edb_wait_states` version 1.2 or later installed and loaded. You must also create the extension,
+preferably on the main database, so the `edb_wait_states` functions are available for Postgres Workload Report to collect saved snapshot data.
```

Comment:
It's not necessary to list a table of contents because our design provides that. As we add content, this becomes unruly (and unnecessary) on the landing page.

Reply:
@ebgitelman I was responding to a comment from @djw-m who suggested that we can't expect users to find subordinate topics by looking at the left nav tree.

Reply:
I agree that it is not necessary.

Reply:
I would like to disagree - relying on the left hand nav bar which for navigation from the landing page of any top index is cognitive load for the reader. I've been applying true index pages in PGD and readers comment immediately on how they can find things more easily. 

That said, I would alter the layout and remove the "More information" presentation and use explanatory lines eg.

* Installing ....